fastly/ngwaf/v1/workspaces/api_create.go

```diff
@@ -0,0 +1,54 @@
+package workspaces
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/fastly/go-fastly/v10/fastly"
+)
+
+// AttackSignalThresholdsInput are the parameters for system site alerts.
+type AttackSignalThresholdsInput struct {
+	OneMinute  *int  `json:"one_minute,omitempty"`
+	TenMinutes *int  `json:"ten_minutes,omitempty"`
+	OneHour    *int  `json:"one_hour,omitempty"`
+	Immediate  *bool `json:"immediate,omitempty"`
+}
```

Comment:
omitting those values to not set go's zero values.

Reply:
I have created separate input structs so that they don't share it anymore: fda74b04a4f88c8a1355fb21bb73a847828deb1d